Backend_Mapanimal/Entities/Race.cs

```diff
@@ -2,8 +2,8 @@
 {
     public class Race
     {
-        public int Id { get; set; }
+        private int Id { get; set; }
```

Comment:
just the setters must be private, in another way we can read values from entities.

Reply:
resolved